tools/pythonpkg/tests/fast/pandas/test_df_object_resolution.py

```diff
@@ -581,6 +581,28 @@ def test_numeric_decimal_varying_sizes(self):
         print(reference)
         print(conversion)
 
+    def test_numeric_decimal_fallback_to_double(self):
```

Comment:
Can you also add tests with a  column with a mix of decimals widths? 
I.e., ones that fit and some that don't